libpod/runtime_img.go

```diff
@@ -313,15 +313,22 @@ func (r *Runtime) LoadImageFromSingleImageArchive(ctx context.Context, writer io
 		func() (types.ImageReference, error) {
 			return layout.NewReference(inputFile, "")
 		},
+		func() (types.ImageReference, error) {
```

Comment:
:1st_place_medal: love the idea!

I think @edsantiago will like it as well; the new error message is so much more user friendly than before.